src/axl.c

```diff
@@ -490,9 +523,10 @@ static kvtree* AXL_Config_Get()
     /* this should be protected by a mutex_lock to prevent issues with
      * realloc() moving memory when growing axl_kvtrees, but no one else
      * does ... */
```

Comment:
You many need to hold the lock here before iterating over the all the kvtrees, as the count could change.